.dictionary.txt

```diff
@@ -1,2 +1,3 @@
+Gitleaks
```

Comment:
Let's also add this to CSpell's software-tools dictionary.

Reply:
I just made a PR for this.